prbt_hardware_support/test/unittests/unittest_sto_modbus_adapter.cpp

```diff
@@ -78,57 +84,68 @@ using ::testing::_;
 class PilzStoModbusAdapterTest : public ::testing::Test, public ::testing::AsyncTest
 {
 protected:
-  void SetUp();
+  virtual void SetUp() override;
+  virtual void TearDown() override;
```

Comment:
Same as above. Why not use constructor/destructor?

Reply:
Done.